examples/java/buildSrc/src/main/groovy/uk.gov.api.spring-boot-conventions.gradle

```diff
@@ -5,7 +5,14 @@ plugins {
 }
 
 configurations {
-  all*.exclude group: 'org.springframework.boot', module: 'spring-boot-starter-logging'
   testImplementation.exclude group: 'org.apache.logging.log4j', module: 'log4j-slf4j-impl'
   testRuntimeOnly.exclude group: 'org.junit.vintage', module: 'junit-vintage-engine'
 }
+
+configurations.all {
+  resolutionStrategy.eachDependency { DependencyResolveDetails details ->
+    if (details.requested.group == 'org.apache.logging.log4j') {
+      details.useVersion '2.16.0'
+    }
+  }
+}
```

Comment:
I'd recommend we manage this through `ext` if possible https://spring.io/blog/2021/12/10/log4j2-vulnerability-and-spring-boot